qiskit/pulse/instructions/__init__.py

```diff
@@ -13,10 +13,12 @@
 # that they have been altered from the originals.
 
 """The ``instruction`` module holds the various ``Instruction`` s which are supported by
-Qiskit Pulse. Instructions each contain operands unique to that instruction, but all have an
-implicit or explicit duration, and a channel on which it is played.
+Qiskit Pulse. Instructions accept a list of operands unique to instructions of that type. Every
+instruction includes at least one :py:class:`~qiskit.pulse.channels.Channel` as an operand
```

Comment:
I'm not positive this will always be true, so perhaps this statement should be relaxed.

Reply:
Is it true currently? If it becomes not true, we will update the documentation, right?

Reply:
It is true currently, but I worry that it sends a message that we require this to be always true to other developers.